sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/SubExprEvaluationRuntime.scala

```diff
@@ -98,7 +96,7 @@ class SubExprEvaluationRuntime(cacheMaxEntries: Int) {
       val proxy = ExpressionProxy(expr, proxyExpressionCurrentId, this)
       proxyExpressionCurrentId += 1
 
-      proxyMap.putAll(e.map(_ -> proxy).toMap.asJava)
+      e.map(proxyMap.put(_, proxy))
```

Comment:
Actually, @viirya. Mind adding some comments that show some examples of the expected seqs and maps here or in the method docs? a rough example like `commonExprs ->  Seq(Seq(common expr 1, ... common expr n), ...)` and `proxyMap -> Map(common expr 1 -> proxy(common expr 1), common expr n -> proxy(common expr 1)`. I think it's easy to make a mistake here.

Reply:
okay.

Reply:
Thanks!